### PR TITLE
Rock and roll

### PR DESCRIPTION
**Pull request by abtrout**
Made some improvements to the rocks, and made the background dark like space.

### CODE REVIEW

File: lib/constants.js [hunk 1]
```diff
@@ -22,6 +22,15 @@ export let constants = {
   // How long a bullet lasts after being fired
   bulletDecay: 2000,
 
+  // Probability that RockMaker creates new rocks.
+  newRockChance: 0.4,
```
Comment (abtrout) [inline lib/constants.js]: I'd like to rename this so that it starts with `rock` for consistency ... `rockMakerChance`?

File: lib/rock.js [hunk 2]
```diff
@@ -9,12 +9,11 @@ export class RockMaker {
   }
 
   start(stage) {
-    // Regularly create new rocks
     setInterval(() => this.maybeMakeRocks(stage), 500)
   }
 
   maybeMakeRocks(stage) {
-    if (Math.random() > this.rocksPct) {
+    if (Math.random() < constants.newRockChance) {
```
Comment (abtrout) [inline lib/rock.js]: Notice I flipped the sign but also changed newRockChance from 0.6 to 0.4.

File: lib/constants.js (hunk 1 above)
Comment (vimarks) [reply lib/constants.js]: SGTM!

File: lib/rock.js (hunk 2 above)
Comment (vimarks) [reply lib/rock.js]: so chance is higher, cause 0 is inclusive and 1 is exclusive?

Comment (icgood) [reply lib/rock.js]: True!

Assuming [0, 1) is represented by 31 bits (1 bit is reserved to indicate positive or negative, and it's always positive), we've increased the chance of a rock by 1/(2^31). Be safe out there.

**Pull request merged**